services/data-collection/autocapture.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Autocapture
+description: Automatically capture events from your website.
+icon: wand-sparkles
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Autocapture allows you to automatically collect various user interactions and events from your website without writing any custom code. 
+By enabling different autocapture settings, you can gather valuable analytics data with minimal setup.
+
+## Configuration
+
+You can configure autocapture settings in your project settings. Each feature can be enabled or disabled independently to match your specific needs.
+
+<Frame caption="Autocapture settings in your project dashboard">
+  <img src="/images/autocapture.png" alt="Autocapture Settings" />
+</Frame>
+
+### Available Features
+
+#### Pageview Tracking
+Automatically captures when users visit your pages. This feature is enabled by default and provides essential analytics about:
+- Page visits
+- Sessions
+- User navigation patterns
+- Entry and exit pages
+
+#### SPA Pageview Tracking
+Specifically designed for Single Page Applications (SPAs), this feature automatically captures page views by intercepting:
+- `history.pushState()` calls
+- `history.replaceState()` calls
+- Browser back/forward navigation (`popstate` events)
+
+<Warning>
+You don't need to enable this feature if you're manually integrating the [Edgee React SDK](/getting-started/react-sdk).
```

Comment:
Does this mean that auto-capture for SPAs doesn't work if you use the React SDK?

Reply:
It works great! But it doubled the pageviews:
- React SDK already launches page event
- Autocapture launches page event as well